src/IndexBuilder.php

```diff
@@ -74,8 +72,10 @@ public function speedUpRefresh(Index $index): void
 
     public function migrate(Index $current, Index $new)
     {
-        // @todo Waiting for https://github.com/ruflin/Elastica/pull/1637 to be merged
-        // This method should use the TASK API, because we do not want to WAIT for the reindex (HTTP Timeout issues).
+        $reindex = new Reindex($current, $new);
```

Comment:
It would be great if this library provide a way to configure how `reindex` will run, I mean choosing a strategy to slice, and to specify a pipeline, or even to run multiples reindex with specific queries each one

... unless the goal is to provide a basic migrate strategy

imHo, this is out of scope of "bump elastica version to ^7", and could be place in another PR

Reply:
Originally I made this PR to use this lib with Elasticsearch 7.3 and so I had to use Elastica ^7. For now it's working well and I start to see some improvement I could make. 
Actually I don't have time to work on it but I would in the next months :)

BTW Great work ! <3

Reply:
Thanks for your feedback, I think we will take over your changes in separated pull requests, so we can merge the Elastica version bump soon :+1: